Makefile

```diff
@@ -13,12 +13,28 @@
 
 all: release
 
+# Improve build speeds during development by removing the --no-cache flag
+DOCKER_BUILD_FLAGS=--no-cache
```

Comment:
so this is just a nicety so if I want to remove `--no-cache` I edit the makefile in one place instead of multiple + script?

Reply:
That's right. you can change all the docker build flags in one place